code/__DEFINES/mobs.dm

```diff
@@ -52,8 +52,8 @@
 /*see __DEFINES/inventory.dm for bodypart bitflag defines*/
 
 //Brain Damage defines
-#define BRAIN_DAMAGE_MILD 50
-#define BRAIN_DAMAGE_SEVERE 120
+#define BRAIN_DAMAGE_MILD 20
```

Comment:
some of the mild ones are actually highly, highly debilitating.

Reply:
Most aren't, brain damage isn't common and it only takes mannitol to cure. Right now traumas are basically opt-in rather than a damage system